code/modules/fields/timestop.dm

```diff
@@ -1,79 +1,90 @@
-
-/obj/effect/timestop
-	anchored = TRUE
-	name = "chronofield"
-	desc = "ZA WARUDO"
-	icon = 'icons/effects/160x160.dmi'
-	icon_state = "time"
-	layer = FLY_LAYER
-	pixel_x = -64
-	pixel_y = -64
-	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
-	var/list/immune = list() // the one who creates the timestop is immune, which includes wizards and the dead slime you murdered to make this chronofield
-	var/turf/target
-	var/freezerange = 2
-	var/duration = 140
-	var/datum/proximity_monitor/advanced/timestop/chronofield
-	alpha = 125
-	var/check_anti_magic = FALSE
-	var/check_holy = FALSE
-
-/obj/effect/timestop/Initialize(mapload, radius, time, list/immune_atoms, start = TRUE)	//Immune atoms assoc list atom = TRUE
-	. = ..()
-	if(!isnull(time))
-		duration = time
-	if(!isnull(radius))
-		freezerange = radius
-	for(var/A in immune_atoms)
-		immune[A] = TRUE
+//THIS DOES NOTHING... FOR NOW.
+GLOBAL_VAR(timestop)
+GLOBAL_LIST_INIT(timestop_blacklist, typecacheof(list(/obj/screen, /obj/effect, /obj/machinery/light, /mob/dead))) // for some reason lightbulbs, and just lightbulbs, act up when timestopped.
+GLOBAL_LIST_INIT(timestop_whitelist, typecacheof(list(/obj/screen/parallax_layer)))
+GLOBAL_LIST_INIT(timestop_noz, typecacheof(list(/obj/screen)))
+
+/proc/get_final_z(atom/A)
+	var/turf/T = get_turf(A)
+	if(T)
+		return T.z
+
+/datum/timestop

```

Comment:
why are you including this in a holopara PR

holoparas don't even get to use this

Reply:
I'm not sure, it seems to me he's trying to sneak in timestop changes which should be in a seperate PR

Reply:
its disabled in the DME file, i can probably remove it but i was using it for a dependency the holoparas DO use

Reply:
ok zoomer